javascript/javascript3/week3/repositoriesPromises/promise.js

```diff
@@ -0,0 +1,32 @@
+// fetching repos of users
+const classmate1 = fetch(
+    "https://api.github.com/search/repositories?q=user:benna100"
+).then(response => response.json());
+const classmate2 = fetch(
+    "https://api.github.com/search/repositories?q=user:islam-fawzy25"
+).then(response => response.json());
+const classmate3 = fetch(
+    "https://api.github.com/search/repositories?q=user:Hani-far"
+).then(response => response.json());
+// promiseall
+function promiseAll() {
```

Comment:
I think better name of the function can be something like getAllRepositories so it is not that general